### DISCUSSION TITLE
How to customize the installdir dialog

### DISCUSSION THREAD

**Discussion by radz1** [wixtoolset/issues#6344] [Questions] [answered]
hi, 
I have InstallDirDlg ui with my msi. 

I need to have one more folder location browse control in the same dialog as below. One for install location and another for log location. I achieved the below UI by customize the InstallDirDlg.wxs file. 

I am not able to set PROPERTY value for the second path for WIXUI_LOGDIR. I need to set both the variable on click of next button- WIXUI_INSTALLDIR, WIXUI_LOGDIR. **WIXUI_INSTALLDIR** variable works fine and got set.

pls suggest.

`<Property Id="WIXUI_LOGDIR" Value="INSTALLFOLDER" />`

`
      
        <Control Id="Next" Type="PushButton" X="236" Y="243" Width="56" Height="17" Default="yes" Text="!(loc.WixUINext)" />
        <Control Id="Back" Type="PushButton" X="180" Y="243" Width="56" Height="17" Text="!(loc.WixUIBack)" />
        <Control Id="Cancel" Type="PushButton" X="304" Y="243" Width="56" Height="17" Cancel="yes" Text="!(loc.WixUICancel)">
          <Publish Event="SpawnDialog" Value="CancelDlg">1</Publish>
        </Control>

        <Control Id="Description" Type="Text" X="25" Y="23" Width="280" Height="15" Transparent="yes" NoPrefix="yes" Text="!(loc.InstallDirDlgDescription)" />
        <Control Id="Title" Type="Text" X="15" Y="6" Width="200" Height="15" Transparent="yes" NoPrefix="yes" Text="!(loc.InstallDirDlgTitle)" />
        <Control Id="BannerBitmap" Type="Bitmap" X="0" Y="0" Width="370" Height="44" TabSkip="no" Text="!(loc.InstallDirDlgBannerBitmap)" />
        <Control Id="BannerLine" Type="Line" X="0" Y="44" Width="370" Height="0" />
        <Control Id="BottomLine" Type="Line" X="0" Y="234" Width="370" Height="0" />

        <Control Id="FolderLabel" Type="Text" X="20" Y="60" Width="290" Height="30" NoPrefix="yes" Text="!(loc.InstallDirDlgFolderLabel)" />
        <Control Id="Folder" Type="PathEdit" X="20" Y="100" Width="320" Height="18" Property="WIXUI_INSTALLDIR" Indirect="yes" />
        <Control Id="ChangeFolder" Type="PushButton" X="20" Y="120" Width="56" Height="17" Text="!(loc.InstallDirDlgChange)" />

        <Control Id="LogFolderLabel" Type="Text" X="20" Y="140" Width="290" Height="30" NoPrefix="yes" Text="Log location" />
        <Control Id="LogFolder" Type="PathEdit" X="20" Y="180" Width="320" Height="18" Property="WIXUI_LOGDIR" Indirect="yes" />
        <Control Id="ChangeLogFolder" Type="PushButton" X="20" Y="200" Width="56" Height="17" Text="!(loc.InstallDirDlgChange)" />
      </Dialog>`

![image](https://user-images.githubusercontent.com/14122410/106171310-6537d780-61b7-11eb-8e58-fb24f294114a.png)

**Comment by BMurri** [comment, marked as answer]
IIRC, each popup window has to be duplicated to make that work (so you call the different one for the other path).

**Reply by radz1** [reply]
Thanks for the reply. I have added separate browsedialog and added event 'setvariable' to set my custom LOGFOLDER variable.
